charts/llamacloud/templates/backend/deployment.yaml

```diff
@@ -48,7 +48,7 @@ spec:
         - name: {{ .Values.backend.name | default "backend" }}
           securityContext:
             {{- toYaml .Values.backend.securityContext | nindent 12 }}
-          image: "{{ .Values.backend.image.repository }}:{{ .Values.backend.image.tag | default .Chart.AppVersion }}"
+          image: "{{ .Values.frontend.image.registry }}/{{ .Values.backend.image.repository }}:{{ .Values.backend.image.tag | default .Chart.AppVersion }}"
```

Comment:
can you rename this to `.Values.<service-name>.image.registry`? each service has its own `<service-name>.image.<values>` block